### ISSUE TITLE
Dropout layer in keras

### ISSUE THREAD

**Issue by cpranav93** [carpentries-incubator/deep-learning-intro#410]
Currently, in unit 4, the value provided to the dropout function is said to be the amount of neurons that are not dropped. However, the documentation of Keras indicates the opposite that the value provided inside dropout is the rate at which neurons are dropped. See [docs](https://keras.io/api/layers/regularization_layers/dropout/). This should be changed.

**Comment by svenvanderburg** [comment]
Yes, I remember that this was confusing during our last teaching.

**Comment by psteinb** [comment]
Just happened to us again.
